src/domain/shared/components/PageHeader/PageBanner.tsx

```diff
@@ -52,14 +52,22 @@ const Ellipser = styled('div')(() => ({
   },
 }));
 
-const Image = styled('img')(() => ({
+const ImageWrapper = styled('div')(() => ({
   position: 'absolute',
   top: 0,
   width: '100%',
   height: '100%',
+  overflow: 'hidden',
   zIndex: 10,
 }));
 
+const Image = styled('img')(() => ({
+  objectFit: 'cover',
+  objectPosition: '0 0',
```

Comment:
@techsmyth Thanks, yes it does. That loading animation behind SDG logo shouldn't be visible anymore, and of course the image is not stretched any more.

We can play with this line, `objectPosition: '0 0',` Setting it to `50% 50%` centers the image and crops the middle, but I think cropping by the top is more consistent